extension/src/contentscript/index.ts

```diff
@@ -1,13 +1,25 @@
-console.log(`Loaded into page`);
-
 import { browser } from "webextension-polyfill-ts";
 
+console.log(`[CS] Loading WatchWithMe Extension content script`);
+
 const csPort = browser.runtime.connect(undefined, { name: "PORT-CS" });
-csPort.postMessage("This is the content script");
+csPort.postMessage({
+    type: "debugMessage",
```

Comment:
The types should be an enum

ex: 
enum Type {
  DEBUG_MESSAGE: "DEBUG_MESSAGE";
}